sdk-src/src/main/java/com/hpe/adm/nga/sdk/Octane.java

```diff
@@ -266,7 +266,11 @@ public Octane build() {
 
         @Override
         public String toString() {
-            return String.format("Server: %s2%s1SharedSpace: %s3Workspace: %s4", System.lineSeparator(), urlDomain, idsharedSpaceId, workSpaceId);
+            return "Builder{" +
```

Comment:
I regenerated the to string method, since it wasn't formatting it nicely.
Didn't pay attention to what it made.
I'll switch it back to format, to be consistent.
But concatenation is faster than format, ofc StringBuilder would be the fastest option.